lib/syskit/test/instance_requirement_planning_handler.rb

```diff
@@ -56,17 +56,21 @@ def apply_requirements
                 )
             end
 
-            def finished?
+            def finished? # rubocop:disable Metrics/AbcSize
```

Comment:
Did we talk about disabling rubocop cops ?

Reply:
I did see this and was planning to resolve it but forgot...thanks for the reminder